src/Components/DrillIllustration.js

```diff
@@ -120,49 +106,6 @@ const DrillIllustration = props => {
     );
   };
 
-  const displayAnimation = ({ illustrationSource, instruction, title }) => {
-    return (
-      <>
-        <Animation widthRatio={1} heightRatio={props.minimal ? 2 / 5 : 1 / 2} animation={illustrationSource} />
-        {props.drill.steps.length > 1 && (
-          <>
-            <View style={styles.description}>
-              <View style={styles.containerAnimation}>
-                <View style={styles.descriptionAnimation}>
-                  <View style={styles.subSubWrapper}>
-                    <Text style={styles.fitness}>{title}</Text>
-                  </View>
-                  <TouchableOpacity style={styles.container} onPress={() => checkAnimationFast()}>
-                    <Animated.View
-                      style={[
-                        {
-                          opacity: opacityUnchecked,
-                        },
-                      ]}
-                    >
-                      <Image style={styles.buttonNext} source={buttonValidation} />
-                    </Animated.View>
-                    <Animated.View
-                      style={[
-                        {
-                          opacity: opacityChecked,
-                        },
-                      ]}
-                    >
-                      <Image style={styles.buttonNext} source={buttonValidationGradient} />
-                    </Animated.View>
-                  </TouchableOpacity>
-                </View>
-              </View>
-            </View>
-            <View style={styles.lines} />
-          </>
-        )}
-        <Text style={styles.instruction}>{instruction}</Text>
```

Comment:
Maybe we should keep this one as long as we d'ont have text linked to animation frames